go.mod

```diff
@@ -4,7 +4,10 @@ go 1.20
 
 require github.com/google/uuid v1.6.0
 
-require github.com/stretchr/testify v1.9.0
+require (
```

Comment:
I would prefer seperate the require lines to oone dependency per line

Reply:
Oops, looks like there is more than one file with the require problem, let me push a new change